exec/memory/base/src/main/java/org/apache/drill/exec/memory/BaseAllocator.java

```diff
@@ -793,20 +797,20 @@ public static boolean isDebug() {
     return DEBUG;
   }
 
-  /**
-   * Disk I/O buffer used for all reads and writes of DrillBufs.
-   */
-
-  private byte ioBuffer[];
-
   public byte[] getIOBuffer() {
     if (ioBuffer == null) {
+      // Length chosen to the smallest size that maximizes
+      // disk I/O performance. Smaller sizes slow I/O. Larger
+      // sizes provide no increase in performance.
+      // Revisit from time to time.
```

Comment:
Fixed. It is a constant because no one can detect changes to performance of the number from outside Drill. To come up with the number required intense, focused tests.